Packs/Gatewatcher/pack_metadata.json

```
@@ -0,0 +1,22 @@
+{
+    "name": "Gatewatcher",
+    "description": "GATEWATCHER is a European leader in advanced Threats detection, protecting critical networks of large Entreprises and Governement organisations since 2015.",
+    "support": "partner",
+    "currentVersion": "0.1.0",
```

For new packs we usually use version 1.0.0:
```suggestion
    "currentVersion": "1.0.0",
```

---

done